src/botPage/view/Dialogs/Limits.js

```diff
@@ -65,8 +82,8 @@ class LimitsContent extends PureComponent {
                                 }}
                                 type="number"
                                 id="limitation-max-loss"
-                                min="0.01"
-                                step="0.01"
+                                step="any"
+                                onChange={this.onChange.bind(this)}
```

Comment:
Although I'd prefer a syntax similar to bind too, but better to keep this consistent with other callback props. `() => this.func()`

Reply:
Done